datafusion/expr/src/expr_fn.rs

```diff
@@ -267,15 +267,21 @@ pub fn approx_percentile_cont_with_weight(
 /// Create an EXISTS subquery expression
 pub fn exists(subquery: Arc<LogicalPlan>) -> Expr {
     Expr::Exists {
-        subquery: Subquery { subquery },
+        subquery: Subquery {
+            subquery,
+            outer_ref_columns: vec![],
+        },
```

Comment:
Maybe we can add `outer_ref_columns` as argument?
I find some tests depend it, and we can cover more test(I can help add this).